rakelib/kokoro/kokoro.rb

```diff
@@ -127,6 +134,27 @@ def check_links location_list, base, tail
     broken_links
   end
 
+  def local_docs_test
+    run "bundle exec rake yard"
+    broken_links = check_links ["doc"], ".", ""
+    puts_broken_links broken_links
+  end
+
+  def should_link_check?
+    return false unless @gem && @should_release
+
+    gem_search = `gem search #{gem}`
+    gem_search.include?(gem)
```

Comment:
Is `gem` supposed to be passed into `should_link_check?`?

Reply:
In the presubmit case, we're iterating over a number of gems, so I assume we need to pass which gem we're querying as an argument to `should_link_check?`